tests/multithreading/thread_local_storage_05.cc

```diff
@@ -34,7 +34,7 @@ Threads::ThreadLocalStorage<X> tls_data;
 
 
 void
-execute(Threads::Mutex &m)
+execute(Threads::Mutex *m)
```

Comment:
Why is this change necessary?

Reply:
Fixed differently, but the issue is that `std::thread` copies its arguments even if they are meant to be references.